modules/arm_plugin/src/CMakeLists.txt

```diff
@@ -21,16 +32,17 @@ ie_add_plugin(NAME ${TARGET_NAME}
               VERSION_DEFINES_FOR arm_plugin.cpp)
 
 target_include_directories(${TARGET_NAME} PRIVATE
-    "${CMAKE_CURRENT_SOURCE_DIR}"
-    "${IE_MAIN_ARM_PLUGIN_SOURCE_DIR}/include")
+        "${CMAKE_CURRENT_SOURCE_DIR}"
+        "${IE_MAIN_ARM_PLUGIN_SOURCE_DIR}/include"
+        "${HEADERS_TRANSFORMATIONS}")
```

Comment:
HEADERS_TRANSFORMATIONS is not needed here

Reply:
@ilya-lavrenov done